### ISSUE TITLE
Custom replacements don't work as expected

### ISSUE THREAD

**Issue by wujekbogdan** [sindresorhus/slugify#9]
... Or at least don't work as I would expect them to work :)

Example:

``` javascript
import slugify from '@sindresorhus/slugify';

const customSlugify = string => slugify(string, {
  customReplacements: [
    ['.', ''],
  ],
});

const slugified = customSlugify('x.y.z'); // Expected `xyz`, received `x-y-z`
```

I expected all dots to be converted into empty strings, so the expected result was `xyz` but I received `x-y-z`

---
I suppose that it's not a bug, but it would be good if the documentation explained how custom replacements really work.

**Comment by nagman** [comment]
Agree with this. I would like to keep the dots for renaming images.

**Comment by ArfatSalman** [comment]
I did some digging and here's what I found - 
In the function `doCustomReplacements`,  this line does the replacement. 
```js
string = string.replace(new RegExp(escapeStringRegexp(key), 'g'), ` ${value} `);
```

Now, if you notice, the second parameter of `replace` has a space, and then the `value` , and another space. 

If you pass an empty string (which you are doing currently) it becomes two spaces.

Later, the spaces become `-`. That's how you get your output of `x-y-z`.

I'll try making a PR to fix this.
